integration/spark-common/src/main/scala/org/apache/carbondata/spark/load/DataLoadProcessBuilderOnSpark.scala

```diff
@@ -179,6 +182,93 @@ object DataLoadProcessBuilderOnSpark {
     updateLoadStatus(model, partialSuccessAccum)
   }
 
+  def insertDataUsingGlobalSortWithInternalRow(sparkSession: SparkSession,
```

Comment:
move `sparkSession: SparkSession,` to next line

Reply:
ok, I follow this guideline. Auto refactor will keep in the same line.